components/dashboard/src/projects/Prebuilds.tsx

```diff
@@ -329,12 +329,66 @@ export function prebuildStatusIcon(prebuild?: PrebuildWithStatus) {
     }
 }
 
+function PrebuildStatusDescription(props: { prebuild: PrebuildWithStatus }) {
+    switch (props.prebuild.status) {
+        case "queued":
+            return <span>Prebuild is queued and will be processed when there is execution capacity.</span>;
+        case "building":
+            return <span>Prebuild is currently in progress.</span>;
+        case "aborted":
+            return (
+                <span>
+                    Prebuild has been cancelled. Either a user cancelled it, or the prebuild rate limit has been
+                    exceeded. {props.prebuild.error}
+                </span>
+            );
+        case "failed":
+            return <span>Prebuild failed for system reasons. Please contact support. {props.prebuild.error}</span>;
+        case "timeout":
+            return (
+                <span>
+                    Prebuild timed out. Either the image, or the prebuild tasks took too long. {props.prebuild.error}
+                </span>
+            );
+        case "available":
+            if (props.prebuild?.error) {
+                return (
+                    <span>
+                        The tasks executed in the prebuild returned a non-zero exit code. {props.prebuild.error}
```

Comment:
**suggestion**: Also, the error appended there is already included in the log output above, right? If yes, I think we could safely drop it from there. WDYT?

<img width="811" alt="Frame 369" src="https://user-images.githubusercontent.com/120486/160592030-c6aa2f31-2ffd-471a-bfce-818ebf96fdfd.png">

Reply:
Definitely. I'm trying to clean it up incrementally as there's a lot of tech debt with respect where we collect the "truth" from. Unify first, then improve the messaging. 

I'll cut a ticket for this concrete case to not show the same error twice.

Reply:
> Unify first, then improve the messaging.

💯 

> I'll cut a ticket for this concrete case to not show the same error twice.

Thanks, @easyCZ! 🏀

Reply:
**fyi**: I've opened https://github.com/gitpod-io/gitpod/issues/9111 to keep track of this discussion and the unresolved comments below.